packages/ubuntu_widgets/CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+## 0.6.0
```

Comment:
Empty changelog here

Reply:
It's because all commits were chores, I'll add  manual entries

Reply:
Thanks, same for localizations and lints

Reply:
Fixed!